Create Leal2-6465.md

Lealdos opened the pull request:
# Create Jose L Delgado Avila profile

## pratice 2 week 1

ntoniocp reviewed (approved):
Test

glrodasz commented on github-profiles/Leal2-6465.md:
```diff
@@ -0,0 +1,6 @@
+# My github profile  https://github.com/Lealdos
```
This file shouldn't exist, because it has been move into the folder instead.

Lealdos replied:
Hello Guillermo, I fixed the error and deleted the leftover file


![image](https://user-images.githubusercontent.com/105816982/224592975-d93564cc-dc16-48c3-9271-eb0e3eeb6297.png)

glrodasz reviewed (changes requested):
Please fix the comments.

glrodasz commented:
I couldn't find the user on Discord it should be `leal2#6465`?

Lealdos commented:
Hello Guillermo, ty for your comments I already fixed them, and in my last commit I added the CSS file with styles for the index.html

Yes, my discord profile is **Leal2#6465**

![image](https://user-images.githubusercontent.com/105816982/224592975-d93564cc-dc16-48c3-9271-eb0e3eeb6297.png)

glrodasz commented:
@Lealdos Excellent, there are just two little details left before merging: 

1. You delete the root README.md file, that one we should keep it.
2. rename your own `readme.md` to `README.md` so we keep the convention.

Lealdos commented:
Hello, @glrodasz README.md created in the root and name file in **lealdos-6465 modified** to UPPERCASES

Lealdos reviewed (commented):
I already did the changes that you asked for
